modules/RTC/JitsiTrack.js

```diff
@@ -310,4 +312,46 @@ JitsiTrack.prototype.getMSID = function () {
     return (streamId && trackId) ? (streamId + " " + trackId) : null;
 };
 
+/**
+ * Set new audio output device for track's DOM elements.
+ * @param {string} audioOutputDeviceId - id of 'audiooutput' device from
+ *      navigator.mediaDevices.enumerateDevices()
+ * @emits JitsiTrackEvents.TRACK_AUDIO_OUTPUT_CHANGED
+ * @returns {Promise}
+ */
+JitsiTrack.prototype.setAudioOutput = function (audioOutputDeviceId) {
+    var self = this;
+
+    if (!RTCUtils.isAudioOutputDeviceChangeAvailable()) {
+        return Promise.reject(
+            new Error('Audio output device change is not supported'));
+    }
+
+    return Promise.all(this.containers.map(function(element) {
+        return element.setSinkId(audioOutputDeviceId)
+            .catch(function (error) {
+                console.error('Failed to change audio output device on element',
+                    element, error);
+
+                // TODO: for some reason 'AbortError' is raised on video
+                // elements with local track blobs. Maybe this is something
+                // similar to https://goo.gl/TKLiqx. Ignoring this error for
+                // now. Spec says that "If the device identified by the given
+                // sinkId cannot be used due to a unspecified error, throw a
+                // DOMException whose name is AbortError."
+                // In any case, all audio communication is done via separate
+                // audio elements, so maybe it doesn't make sense to change
+                // sinkId for <video> elements at all.
+                if (!(self.isVideoTrack() && self.isLocal && self.isLocal() &&
```

Comment:
Actually found the reason why this happens - when media stream from GUM has only video tracks and no audio tracks. In RTCUtils handleLocalStream function we actually split stream obtained from GUM into separate audio and video streams. So I suggest just to ignore elements for local tracks at all in this case. And maybe even all 'video' elements can be ignored as audio communication is done via 'audio' elements?